### PR TITLE
Create api endpoints

### PR DESCRIPTION
**Pull request by fabrgu**
Create api endpoints for opportunities, requests and offers

### CODE REVIEW

File: api/app.py [hunk 1]
```diff
@@ -15,12 +18,138 @@ def get_users():
...
+    return resp
+
+
+@app.route('/api/request', methods=['POST'])
+def create_update_request():
+
+    content = request.get_json()
+    request_id = content['request_id']
+    requester_user_id = content['requester_user_id']
+    volunteer_user_id = content['volunteer_user_id']
```
Comment (elleneub) [inline api/app.py:79]: I think we won't have this when we first post the request

**Review by elleneub — COMMENTED** [review]
Looks really good! I'm deferring full review to the backend team, but other than the one comment this looks solid.

File: api/app.py (hunk 1 above)
Comment (fabrgu) [reply api/app.py:79]: Yeah, I agree with the volunteer_user_id. We should have the requester_user_id though, right (whoever is creating the request).

Comment (elleneub) [reply api/app.py:79]: Yup! So basically, I would just have the UI not send it and y'all can set to null on your end.

**Review by javajael — COMMENTED** [review]
Looks good - Thanks for doing this Fabiola!

**Review by javajael — APPROVED** [review]

**Pull request merged**